system/usbmsc/Kconfig

```diff
@@ -18,12 +18,21 @@ if SYSTEM_USBMSC
 
 config SYSTEM_USBMSC_NLUNS
 	int "Number of LUNs"
-	default 1
+	default 8
 	---help---
 		Defines the number of logical units (LUNs) exported by the USB
 		storage driver.  Each LUN corresponds to one exported block driver
 		(or partition of a block driver).  May be 1, 2, or 3.  Default is 1.
 
+config SYSTEM_USBMSC_DEVPATH1
+	string "LUN1 Device Path"
+	default ""
+	---help---
+		The full path to the registered block driver.  Default is
+		"/dev/mmcsd0"
+
+if SYSTEM_USBMSC_DEVPATH1 != ""
+
 config SYSTEM_USBMSC_DEVMINOR1
```

Comment:
let's use the special value of SYSTEM_USBMSC_DEVPATH1 to skip LUN1

Reply:
Ditto, as type of `CONFIG_SYSTEM_USBMSC_DEVPATH1` is string, zero length string may be already special enough.